apps/src/templates/courseOverview/AssignToSection.js

```diff
@@ -154,7 +159,8 @@ class AssignCourse extends Component {
         {dropdownOpen && (
           <div style={styles.dropdown}>
             <a
-              href={`${window.dashboard.CODE_ORG_URL}/teacher-dashboard?newSection=${courseId}#/sections`}
+              href={`${window.dashboard.CODE_ORG_URL}/teacher-dashboard?` +
+                queryString.stringify({courseId, scriptId}) + "#/sections"}
```

Comment:
hmm...we should maybe centralize around one? The benefit of this one is it doesn't require redux, but we might have redux enough places now that this doesnt matter